airgun/entities/host.py

```diff
@@ -223,6 +223,13 @@ def get_webconsole_content(self, entity_name):
         view = self.navigate_to(self, 'Details', entity_name=entity_name)
         view.webconsole.click()
         view.validations.assert_no_errors()
+        # switch to the last opened tab,
+        self.browser.selenium.switch_to.window(self.browser.selenium.window_handles[-1])
```

Comment:
You can use widgetastic's browser methods here to deal with window handles, you shouldn't have to use selenium directly.